### DISCUSSION TITLE
请问测试题是否可以在提交答案后有解析？

### DISCUSSION THREAD

**Discussion by BinksGao** [AmazingAng/WTF-Solidity#237] [Q&A] [answered]
请问测试题是否可以在提交答案后有解析？方便自己排查错误。
![image](https://user-images.githubusercontent.com/18161171/194501452-260ebd34-ea04-44c6-b183-f49401553103.png)

**Comment by AmazingAng** [comment, marked as answer]
我们在内测时尝试过显示错误答案，但每一讲题并不多，这样学员太容易通过排除法通过测试。
因此现在不显示了，建议多读几遍教程，找出自己错误。

**Reply by jlulxy** [reply]
有的题感觉教程里面并没有写哇

**Reply by GuldanChen** [reply]
如果已经一百分了是不是应该拿出分析啊，这样也可以加强一下啊

**Comment by Loius0000** [comment]
还一个问题，极简入门数值类型quiz不管答案是对是错怎么做都是80分，是不是有BUG？？？

**Reply by Faye-11** [reply]
我也是，我都怀疑自己了

**Reply by bakmak** [reply]
我也遇到了这个问题，无论如何尝试。它都没有改变过最初的分数...

**Reply by baizetianxia** [reply]
我也是，到底是哪题有问题，或是我们理解不到位

**Reply by Freshield** [reply]
同

**Comment by Loius0000** [comment]
可以不给解析，但起码要告知哪个对了哪个错了吧。解析可以自己研究。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年11月6日(周日) 晚上10:26
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [AmazingAng/WTF-Solidity] 请问测试题是否可以在提交答案后有解析？ (Discussion #237)




 
同样问题

**Reply by AmazingAng** [reply]
https://github.com/AmazingAng/WTF-Solidity/discussions/260
其中解释了：我重新提交了作业，有错题，但没有显示错误选项/原因。

**Comment by Loius0000** [comment]
可以



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年11月7日(周一) 下午2:17
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [AmazingAng/WTF-Solidity] 请问测试题是否可以在提交答案后有解析？ (Discussion #237)




 
#260
 其中解释了：我重新提交了作业，有错题，但没有显示错误选项/原因。
